sdk/src/Services/DynamoDBv2/Custom/DataModel/BatchWrite.cs

```diff
@@ -200,6 +200,23 @@ internal BatchWrite(DynamoDBContext context, DynamoDBFlatConfig config)
             DocumentBatch = table.CreateBatchWrite();
         }
 
+        internal BatchWrite(DynamoDBContext context, Type valuesType, DynamoDBFlatConfig config)
+            : base(context, config)
+        {
```

Comment:
Since the other constructor contains almost the same code, can you update it to call this one? maybe something like this
```
        internal BatchWrite(DynamoDBContext context, DynamoDBFlatConfig config)
            : this(context, typeof(T), config)
        {
        }
```